app/models/post.rb

```diff
@@ -0,0 +1,12 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
```

Comment:
- [ ]  in the Post class the has_many side of the one-to-many relationship between posts and like should be set as well.